run

```diff
@@ -4,10 +4,10 @@ set -eo pipefail
 
 ###
 # Check "ODO_S2I_DEPLOYMENT_DIR" environment variable and if it's present,
-# copy content from /opt/app-root/backup directory to "ODO_S2I_DEPLOYMENT_DIR"
+# copy content from ${ODO_DEPLOYMENT_BACKUP_DIR} directory to "ODO_S2I_DEPLOYMENT_DIR"
 # Ref: https://github.com/redhat-developer/odo/issues/445
-if [ -d /opt/app-root/backup ] && [ -n "$ODO_S2I_DEPLOYMENT_DIR" ]; then
-    cp -rf /opt/app-root/backup/. ${ODO_S2I_DEPLOYMENT_DIR}/
+if [ -d ${ODO_DEPLOYMENT_BACKUP_DIR} ] && [ -n "$ODO_S2I_DEPLOYMENT_DIR" ]; then
+    rsync -r ${ODO_DEPLOYMENT_BACKUP_DIR}/. ${ODO_S2I_DEPLOYMENT_DIR}/
```

Comment:
it would be better to use `rsync -a` to make sure that it is really identical copy

Reply:
Done